src/App.js

```diff
@@ -0,0 +1,64 @@
+        { id: 2, isRented: false, title: "Beauty and the Beast", year: 1991, img: "https://images-na.ssl-images-amazon.com/images/I/81etFyb9N-L._SL1500_.jpg", descrShort: "A kickass woman named Belle who does not succumb to social norms gets crap from a bunch of village idiots, chief amongst them a total tool named Gaston. Belle shows everyone how great she is when she turns a beast (not Gaston) into a man. Love ensues, but then the villagers fall trap to severe group-think mentality led by the main tool himself." },
+        { id: 3, isRented: false, title: "The Sword in the Stone", year: 1963, img: "https://www.disneyinfo.nl/images/laserdiscs/229-1-AS-front.jpg", descrShort: "Arthur is a young boy who just wants to be a knight's squire. Alas, he is dubbed 'Wart' early on, and it was all downhill from there for a while. On a hunting trip he falls in on Merlin, literally. Merlin is a possibly-mentally-unstable-and-ethically-dubious Wizard that turns Arthur into a literate, at-one-point harassed squirrel. Watch to find out what the heck that means." },
+        { id: 4, isRented: false, title: "Beauty and the Beast", year: 2016, img: "https://images-na.ssl-images-amazon.com/images/I/51ArFYSFGJL.jpg", descrShort: "Basically the same as the original, except now Hermi-- Emma Wattson plays Belle, fittingly so some would say, given how actively progressive she is regarding women's rights. Rumor has it that in the bonus scenes she whips out a wand and turns Gaston into a toad, but in order to watch those scenes you need to recite a certain incantation." }
+      ],
+
+      searchVal: '',
+    }
+
+  }
+
+  handleSearch = (v) =>this.setState({searchVal: v})
+
+  rentOrNot = id => {
+    let allMovies = [...this.state.moviesDetails]
+    allMovies.find(m => m.id === id).isRented = !allMovies.find(m => m.id === id).isRented 
+    this.setState({
+      moviesDetails: allMovies
+    })
+  }
+
+  checkRented = movies =>{
+    let rented = []
+     movies.forEach(m => { if(m.isRented) rented.push(m) })
+    if(rented.length === 0) return false
+    else return rented
+  }
+
+  render() {
+    return (
+      <Router>
+        <div className="App">
+          <div id='home-background'>
+            <div id='main-links'>
+              <Link to='/' > <span>Home</span>    </Link>
+              <Link to='/Catalog' > <span>Catalog</span>  </Link>
+              <div id='logo' >REFLIX</div>
+            </div>
+            <Route path='/' exact component={Home} />
+            <Route path='/Catalog' exact render={() => <Catalog key={'catalog'} search={this.handleSearch} method={this.rentOrNot} searchVal={this.state.searchVal} movies={this.state.moviesDetails} rentedMovies={this.checkRented(this.state.moviesDetails)}/>} />
```

Comment:
this component does not need a key. You are not mapping over it and making multiple. only if you use .map() do you need to add keys